src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,16 @@
-export const TodoList = () => {};
+import { TodoInfo } from '../TodoInfo';
+import { ToDo } from '../../types/types';
+
+type Props = {
```

Comment:
You've already started nicely to export typing to separate files, so it would be cool to continue here as well, and export the type for the `TodoList` component to the types.tsx file and give this type a more convenient name (For example: `TodoListType`).